### DISCUSSION TITLE
Missing plots

### DISCUSSION THREAD

**Discussion by Epemba1978** [Chia-Network/chia-blockchain#8151] [Support]
Change the cabinet of my pc, to be able to place more disks. I spent several hours with the computer turned off. Upon restarting the PC, I waited for syncing and then discovered that one of my 4TB drives had only 7 plots when it used to have 36. I don't know what happened to the other 29 plots. The disk is full but only 7 plot files are visible inside.

**Comment by BrandtH22** [comment]
@Epemba1978 , since this ticket was opened a number of years ago without further comments it seems this issue has been resolved. 
We will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions or comments.

Note - The chia client itself does not delete plots, I would verify in a file explorer that the plot files are still on the drive and if not I would check to see what is taking up the remaining data on the drive. It could very well be that the drive did not properly mount so a reboot of the system might also be beneficial.

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
